FatmaQunnies/app/route.py

```diff
@@ -0,0 +1,73 @@
+from FatmaQunnies.classes import urgent_task
+from fastapi import FastAPI
+import uuid
+from classes import *
+from classes.model import TaskModel, TaskUpdate
+
+
+app = FastAPI()
+tasks = {}
+
+@app.post("/tasks/")
+def create_task(task: TaskModel):
+    task_id = str(uuid.uuid4())  
+    if task.priority:
+        new_task = urgent_task(task.title, task.description, task.due_date, task.priority)
+    else:
+        new_task = {
+            "id": task_id,
+            "title": task.title,
+            "description": task.description,
+            "due_date": task.due_date,
+            "status": task.status,
+            "priority": task.priority
+        }
+    tasks[task_id] = new_task 
+    return new_task
+
+
+@app.put("/tasks/{task_id}/")
+def update_task(task_id: str, task_update: TaskUpdate):
+    task = tasks.get(task_id) 
+    if task is None:
+        raise Exception
+
+    if task_update.title is not None:
```

Comment:
What happens if the attributes are more? 10? 20?
are u sure to use this way?
let us discuss the problem: you do the same operations for all properties for an object, I think it is better to use a loop to iterate over all of them and handle these operations.